main.tex

```diff
@@ -864,6 +866,11 @@ \subsection{The effect of population size}\label{sec:population_size}
     \item The zero determinant strategies.
 \end{itemize}
 
+The results for \(x_{N/2}\) show similarities to the results for \(x_{N-1}\) and
+in particular TF1, TF2 and TF3 all rank in the top 8 strategies. This is to be
```

Comment:
I'd say ranked 1, 3, and 8

and

initial population of \((N/2, N/2)\)

Otherwise LGTM